reduxpractice/src/App.js

```diff
@@ -0,0 +1,56 @@
+import logo from './logo.svg';
+import './App.css';
+import Main from './components/MainComponent'
+import Create from './components/CreateComponent';
```

Comment:
import sequence should be

react imports
any packages related imports
shared folders which are in the same app
utils and constants
styles
Fix all the imports in your project